### DISCUSSION TITLE
Unable to Find API for change/Update password?

### DISCUSSION THREAD

**Discussion by Gk-rohan** [kanboard/kanboard#5380] [Q&A] [answered]
Can anyone help me find way to change/update password for user?
I've looked up on previous issues but didn't get any help.

**Comment by fguillot** [comment]
https://docs.kanboard.org/v1/admin/cli/#reset-user-password

**Reply by Gk-rohan** [reply]
Yes I've looked it for. Is there any other way around API?

**Reply by fguillot** [reply, marked as answer]
Not currently. The existing API needs to be extended: https://docs.kanboard.org/v1/api/user_procedures/#updateuser
